packages/apps-config/src/ui/logos/index.ts

```diff
@@ -136,12 +136,13 @@ export const chainLogos: Record<string, unknown> = [
   ['Darwinia PC2', nodeDarwinia],
   ['DataHighway', nodeDataHighway],
   ['Dusty', chainDusty],
-  ['EquilibriumTestnet', chainGenshiro],
+  ['EquilibriumTestnet', chainEquilibrium],
```

Comment:
This has also changed in master now - so will be a small (easily solvable) conflict.